dagger/utilities/module.py

```diff
@@ -14,7 +14,7 @@
 
 
 class Module:
-    def __init__(self, path_to_config, target_dir):
+    def __init__(self, path_to_config, target_dir, jinja_parameters):
```

Comment:
should we make `jinja_parameters` as optional?

Reply:
It is optional. The default=None parameter makes it optional.

Reply:
Ah sorry you are right, in the module I also should have give a default value

Reply:
Fixed it